readthedocs/search/parse_json.py

```diff
@@ -93,24 +84,27 @@ def process_file(fjson_filename):
         log.info('Unable to index title for: %s', fjson_filename)
 
     return {
-        'headers': process_headers(data, fjson_filename),
-        'content': body_content,
         'path': path,
         'title': title,
         'sections': sections,
     }
 
 
-def recurse_while_none(element):
+def parse_content(content):
     """
-    Traverse the ``element`` until a non-None text is found.
-
-    :param element: element to traverse until get a non-None text.
-    :type element: pyquery.PyQuery
+    Removes the starting text and ¶.
 
-    :returns: the first non-None value found
-    :rtype: str
+    It removes the starting text from the content
+    because it contains the the title of that content,
+    which is redundant here.
     """
-    if element.text is None:
-        return recurse_while_none(element.getchildren()[0])
-    return element.text
+    content = content.replace('¶', '').strip()
+
+    # removing the starting text of each
+    content = content.split('\n')
+    if len(content) > 1:  # there were \n
+        content = content[1:]
+
+    # converting newlines to ". "
+    content = '. '.join([text.strip() for text in content])
```

Comment:
Same here

Reply:
What should be the approach here?
I think, we can do `'\n'.join([text.strip() for text in content]` here and then replace new lines when finally yeilding from `parse_json()`?

Reply:
not sure how this could end up with additional `\n`, we should write a test and fix that here. We can just do
 `'. '.join([text.strip().rstrip('.') for text in content])`